src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -45,6 +46,10 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 
 	const { state, dispatch } = useForm();
 
+	const chipListRef:{ current?: HTMLDivElement } = useRef();
+
+	const chipListHeight: number = chipListRef?.current?.offsetHeight ? chipListRef?.current?.offsetHeight + 30 : 0

```

Comment:
Please make sure you finish all your lines with a semicolon (;). In regular JavaScipt it might not be necessary or an issue, but with TypeScript, since it compiles the application, in several cases it can lead to pieces of code not working as expected.